### ISSUE TITLE
BUG: Drag on on long press not working

### ISSUE THREAD

**Issue by TheTushar2696** [hoangnm/react-native-week-view#272]
**Describe the bug**
I am trying to use the dragEventConfig={DRAG_EVENT_CONFIG} to enable drag only on long press, but seems like I am unable to drag when I provide afterLongPressDuration.

I've installed react-native-gesture-handler : v2.6.0
Attaching snap for the same

https://user-images.githubusercontent.com/40525668/192934668-e023af4c-fb3c-42c5-8e89-b0aa26109371.mp4

**Comment by pdpino** [comment]
Could you please:

* show the code (including `DRAG_EVENT_CONFIG` provided)
* check react-native-week-view version you are using

**Comment by TheTushar2696** [comment]
Hi @pdpino 
This is the dragConfig
- <img width="344" alt="Screenshot 2022-09-29 at 10 36 10 PM" src="https://user-images.githubusercontent.com/40525668/193094346-c2c8071d-daa5-4a09-b4aa-4b9f453f4f73.png">
- react-native-week-view version is 0.24

**Comment by pdpino** [comment]
@TheTushar2696 the dragConfig feature will be available in 0.25 (to be released soon)

For now you can try the feature if you install directly from the master branch

**Comment by TheTushar2696** [comment]
Hi @pdpino 
I am currently i am currently installing it from the master branch only, and I could see the code for drag config in the package getting installed.But its not working for me
<img width="639" alt="Screenshot 2022-09-29 at 11 20 24 PM" src="https://user-images.githubusercontent.com/40525668/193105664-7f0e50d8-8c2c-4c16-b7c4-ee87ce322ded.png">

**Comment by pdpino** [comment]
@TheTushar2696 you're right, I checked back the video and I see the code is properly installed. When the event gets further reduced in opacity --> indicates "dragging" mode is activated

I'll check this further and let u know

**Comment by TheTushar2696** [comment]
hi @pdpino 
Any update on the above

**Comment by pdpino** [comment]
I haven't been able to reproduce this issue yet
